docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.icons.scss

```diff
@@ -94,10 +94,21 @@
 }
 
 @mixin hb-icon-link-arrow($color: 'black') {
-  @include hb-icon-background('arrow-right', $color);
   background-repeat: no-repeat;
   background-position: left;
   background-size: 55%;
+
+  @if ($hb-current-theme == 'airy') {
```

Comment:
Are we unable to use our mixins here?

Reply:
lol for some reason I thought I couldnt use mixins in a mixin... even though this is already doing that